src/pages/ContactPage.jsx

```
@@ -1,8 +1,378 @@
+import React, { useState } from 'react';
+import { useToast } from '../hooks/useToast';
+import axiosInstance from '../libs/axios';
```

ESLint

[error] 'axiosInstance' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)